### PR TITLE
Fix navigator layer links for mobile and ICS

### PR DESCRIPTION
**Pull request by mehaase**
Followup to previous commit that fixed layer links for enterprise, this one fixes mobile and ICS links.

### CODE REVIEW

File: mappings/veris-1.3.7/layers/ics/README.md
Comment (tiffb) [inline mappings/veris-1.3.7/layers/ics/README.md:1]: Should "enterprise" be "ics" here?

File: mappings/veris-1.3.7/layers/mobile/README.md
Comment (tiffb) [inline mappings/veris-1.3.7/layers/mobile/README.md:1]: Should "enterprise" be "mobile" throughout this file?

**Review by tiffb — COMMENTED** [review]
Comments on ICS and Mobile READMEs

File: mappings/veris-1.3.7/layers/ics/README.md
Comment (mehaase) [reply mappings/veris-1.3.7/layers/ics/README.md:1]: Yes, good catch. I'll investigate why it's doing that...

Comment (mehaase) [reply mappings/veris-1.3.7/layers/ics/README.md:1]: Should be fixed in latest commit

File: mappings/veris-1.3.7/layers/mobile/README.md
Comment (mehaase) [reply mappings/veris-1.3.7/layers/mobile/README.md:1]: Should be fixed in latest commit

**Review by tiffb — APPROVED** [review]

**Pull request merged**